theano/gof/tests/test_destroyhandler.py

```diff
@@ -47,6 +47,10 @@ def MyConstant(data):
 
 class MyOp(Op):
 
+    __props__ = ("nin", "name", "vmap", "dmap", "nout",
+                 "destroyhandler_tolerate_same",
+                 "destroyhandler_tolerate_aliased")
+
```

Comment:
This part caused the travis build fail. The error was, 
```
======================================================================
ERROR: theano.gof.tests.test_destroyhandler.test_misc
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/Ramana/projects/macvnev/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/tests/test_destroyhandler.py", line 155, in test_misc
    PatternOptimizer((transpose_view, (transpose_view, 'x')), 'x').optimize(g)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/opt.py", line 83, in optimize
    self.add_requirements(fgraph)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/opt.py", line 2170, in add_requirements
    fgraph.attach_feature(toolbox.NodeFinder())
  File "/Users/Ramana/projects/theano/Theano/theano/gof/fg.py", line 564, in attach_feature
    attach(self)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/toolbox.py", line 439, in on_attach
    Bookkeeper.on_attach(self, fgraph)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/toolbox.py", line 124, in on_attach
    self.on_import(fgraph, node, "on_attach")
  File "/Users/Ramana/projects/theano/Theano/theano/gof/toolbox.py", line 464, in on_import
    raise e
AttributeError: 'MyOp' object has no attribute 'vmap'
```
I didn't see it in detail then. The object was of `dict` type and that's why probably.